src/nipple.js

```diff
@@ -118,13 +120,17 @@ Nipple.prototype.stylize = function () {
         'opacity': '.5'
     };
 
+    
+    var nipplesizemultiple = (this.options.nipplesize / 100);
```

Comment:
I think I'd prefer having a number between `0-1` for the `nipplesize` option.
It's more consistent with how we use percentage in prog.
And it will remove this calculation.

What do you think?